src/coprocessor/codec/data_type/vector.rs

```diff
@@ -469,24 +464,6 @@ macro_rules! impl_as_slice {
                     ),
                 }
             }
-
-            /// Extracts a mutable slice containing the entire values of the column
-            /// in specific type.
-            ///
-            /// # Panics
-            ///
-            /// Panics if current column is does not match the type.
-            #[inline]
-            pub fn $mut_name(&mut self) -> &mut [Option<$ty>] {
```

Comment:
Don't we need it anymore?

Reply:
Yes. We don't need a mutable slice (to mutate the content) because our function evaluation never mutates existing data, instead it produce new ones. Additionally, supporting mutating data is not efficient in future data structures (for example, arena string vector). So let's completely remove this feature so that our life is easier in future.